learn/framework/xcm.md

```diff
@@ -0,0 +1,73 @@
+
+### Remote Execution {: #remote-execution }
+
+Another common use is remote execution, which is sending a message to an appchain and triggering some action. If the destination chain is an EVM-compatible one, this allows appchain A to call a smart contract deployed on appchain B, for example.
+
+## Establishing Cross-Chain Communication {: #channel-registration }
+
+Before two chains can start communicating, a messaging channel must be opened. Channels are unidirectional, meaning a channel from chain A to chain B will only pass messages from A to B. Therefore, two channels must be opened to send messages back and forth.
+
+A channel between the appchain and the relay chain is automatically opened upon appchain registration and onboarding. However, when appchain A wants to open a communication channel with appchain B, appchain A must send an open channel transaction to its network. This transaction is an XCM message as well!
+
+It's important to note that appchain A expressing its intentions of opening an XCM channel with appchain B is not sufficient. Appchain B must also accept the request by sending another XCM message. Only when both appchains have reached a mutual agreement, the channel is opened within the following epoch. The same process must be repeated to open a channel from appchain B to appchain A.
+
+![XCM Channel Registration Overview](/images/learn/framework/xcm/dark-xcm-1.webp#only-dark)
+![XCM Channel Registration Overview](/images/learn/framework/xcm/light-xcm-1.webp#only-dark#only-light)
+
+Once the channel is established, cross-chain messages can be sent between appchains. For asset transfers, assets will also need to be registered before being transferred.
+
+## Message Destinations {: #message-destinations }
+
+Any given appchain contains modules exposing behavior through transactions or smart contracts that can be called, accounts with balances, and assets (fungibles or non-fungibles) that can be created, transferred, etc. Also, all the appchains within the ecosystem are L1 blockchains connected to an L0 blockchain, the relay chain, which plays a crucial role in the cross-chain messaging process, with the ability to relay messages from one appchain to another.
+
+To send and receive cross-chain messages that accurately reference the destination chain, its exposed behavior (such as smart contracts or transactions), and its available assets, a mechanism called *Multilocation* is used. Multilocations describe an element from the sender's perspective, so they are not intended to provide a universal description of the element.
+
+Two components define a location: `parents` and `interior`. Parents is a value that indicates if the route must move up to the relay chain. Interior is a list of junctions, that define how to locate the destination. Let's list some examples:
```

Comment:
Might be worth noting here before diving into the examples that multilocations are hierarchical structures. So that is why an "upward" move is necessary to get from a parachain to the relay chain

I prefer not to use "let's"

Reply:
Love this examples section though! 😍